test/Lazy/split.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { split } from "../../src";
+
+describe("split", function () {
+  describe("sync", function () {
+    it("should return an empty array", function () {
+      const iter = split("", "");
+      expect([...iter]).toEqual([]);
+    });
+
+    it("should be splited by empty string", function () {
```

Comment:
Is the word "empty" right?

Reply:
`""` <- This value entered the separator value so I wrote this expression. 
If you have a better expression, please share it :)

Reply:
Understand. No problem.